frontend/src/e2e-test/pages/employee/child-information.ts

```diff
@@ -569,6 +569,36 @@ export class AssistanceNeedSection extends Section {
   async assertAssistanceNeedCount(expectedCount: number) {
     await waitUntilEqual(() => this.#assistanceNeedRow.count(), expectedCount)
   }
+
+  async waitUntilAssistanceNeedDecisionsLoaded() {
+    await this.page
+      .findByDataQa('table-of-assistance-need-decisions')
+      .waitUntilVisible()
+  }
+
+  assistanceNeedDecisions(nth: number) {
```

Comment:
This pattern of returning an object containing promises is a bit problematic, because the calling side has no real control of *when* the information is fetched, and the promises in the returned object must not be used more than once (unless you really want the same cached value multiple times).

I can think of two better choices:

1. await each of the fields in this function, so the returned object is a simple data object
2. return a wrapper class instead, and make getter functions for each of the fields so that each call to a getter returns a new promise